kibbeh/src/ui/Search/SearchBar.tsx

```diff
@@ -5,20 +5,26 @@ import { Input } from "../Input";
 export interface SearchBarProps
   extends React.ComponentPropsWithoutRef<"input"> {
   inputClassName?: string;
+  mobile?: boolean;
 }
 
 export const SearchBar: React.FC<SearchBarProps> = ({
   className = "",
   inputClassName = "",
+  mobile = false,
   ...props
 }) => {
   return (
     <div
-      className={`w-full bg-primary-700 text-primary-300 focus-within:text-primary-100 rounded-lg ${className}`}
+      className={`w-full bg-primary-700 text-primary-300 focus-within:text-primary-100 rounded-lg ${
+        mobile ? "px-4" : ""
+      } ${className}`}
```

Comment:
Hey there! I think this component might be easier to use if instead of passing in a `mobile` prop, you use tailwinds built in stuff for checking for mobile screens. 

```suggestion
      className={`w-full bg-primary-700 text-primary-300 focus-within:text-primary-100 rounded-lg px-4 md:px-0 ${className}`}
```

This way it "automatically" knows it's on a phone vs desktop